ocaml/xapi/message_forwarding.ml

```diff
@@ -1099,6 +1099,12 @@ functor
           value ;
         Local.Pool.set_uefi_certificates ~__context ~self ~value
 
+      let set_custom_uefi_certificates ~__context ~self ~value =
+        info "Pool.set_custom_uefi_certificates: pool='%s' value='%s'"
```

Comment:
Just in case, the certificates printed here don't usually contain private keys, do they?